docs/advanced/api.mdx

```diff
@@ -130,13 +130,15 @@ const visit = require('unist-util-visit')
 const remove = require('unist-util-remove')
 const yaml = require('yaml')
 
-const file = vfile(`
+const file = vfile(
+  `
 ---
 title: Hello, MDX
 ---
 
-I <3 Markdown and JSX
-`)
+I &lt;3 Markdown and JSX {/* < must be escaped for \`@mdx-js/mdx@2\` */}
+`.trimStart() // `trimStart` is required for `remark-frontmatter@3`
```

Comment:
```suggestion
`.trimStart() // there should be no spaces before frontmatter for `remark-frontmatter@3`, this can be guaranteed with `trimStart`
```

@ChristianMurphy How about this?

Reply:
looks good to me :+1: